wled00/cfg.cpp

```diff
@@ -1240,4 +1240,4 @@ void serializeConfigSec() {
   if (f) serializeJson(root, f);
   f.close();
   releaseJSONBufferLock();
-}
+}
```

Comment:
Not essential to revert, but I recommend that Devs always double check the git diff --staged before commit or git show before push to ensure they are only making their intended change. If you use your graphical tools is far too easy to just click buttons and overlook exactly which files it's staging and assume the contents are accurate - treated like a "save all"